components/controller_host.py

```diff
@@ -1,6 +1,7 @@
 from qunetsim.components import Host
 from utils import DefaultOperationTime
 
+
```

Comment:
Ah... sorry my IDE auto formatting this file, but I just changed line 91, small syntax issue